src/util/api-request.ts

```diff
@@ -1,35 +1,20 @@
 import { Cleaner } from 'cleaners'
-import { EdgeFetchFunction, EdgeLogMethod } from 'edge-core-js'
+import { EdgeFetchFunction } from 'edge-core-js'
 import nodeFetch from 'node-fetch'
 
 import { ApiRequest, ApiResponse, asApiResponse } from '../types'
 
-type CustomLog = EdgeLogMethod & {
-  readonly warn: EdgeLogMethod
-  readonly error: EdgeLogMethod
-}
-
 export interface CommonOptions {
   fetch?: EdgeFetchFunction
-  log?: CustomLog
+  log?: (...args: any[]) => void
```

Comment:
Could even be `(message: string) => void` for a truly minimal solution, but this works too.

Reply:
That would restrict us too much from passing objects and what not within our implementations here. Most log functions should accept objects and multiple args.